src/Components/App/App.js

```diff
@@ -53,23 +61,31 @@ export const App = () => {
 
   return (
     <div className="App">
-      <Route exact path='/' render={ () =>
+      <Route exact path='/' render={() =>
         (activeBook.id &&
```

Comment:
This is good advice, thank you. I was planning on adding a some instructions to the 'blank' screen, but moving the conditional rendering would be a good move too. This was a temporary fix to avoid errors while developing, but I hadn't thought through where the real logic should live.